luthier/config/src/config.lua

```diff
@@ -30,7 +30,14 @@ local tableConfig = tableUtils.build_table_config(tables)
 -- a script since creating a dir in Lua is awkard, in future.
 -- can be circumvented by using the LuaFileSystem module
 os.execute("mkdir -p ../external/")
+os.execute("mkdir -p ../../src/test/resource/")
 parser.save("../external/DimensionConfig.json", dimensionConfig)
 parser.save("../external/MetricConfig.json", metricConfig)
 parser.save("../external/TableConfig.json", tableConfig)
 -- parser.save("../external/MakerConfig.json", require("makers"))
+
+-- add to the test/resource
+parser.save("../../src/test/resource/DimensionConfig.json", dimensionConfig)
+parser.save("../../src/test/resource/MetricConfig.json", metricConfig)
+parser.save("../../src/test/resource/TableConfig.json", tableConfig)
+-- parser.save("../../src/test/resource/MakerConfig.json", require("makers"))
```

Comment:
Why is this commented out?

Reply:
This is commented out because we don't have the requirement "maker.lua" yet. 
We can uncomment that when we get there.